test/sequential/sequencerStaking.rules.test.ts

```diff
@@ -381,6 +393,150 @@ describe("sequencerStaking", () => {
     expect(otherSequencerStatus.cancelRights.toString()).toBe("0");
   });
 
+  it("When a cancel resolution fail, maintenance mode will be triggered automatically", async () => {
+    let maintenanceStatus: any;
+    const foundationMembers = await FoundationMembers.getFoundationMembers();
+    const chain = "Ethereum";
+    const sequencer = await setupASequencer(chain);
+    const api = getApi();
+    const txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
+    await Sudo.batchAsSudoFinalized(
+      Sudo.sudoAsWithAddressString(
+        foundationMembers[0],
+        Maintenance.switchMaintenanceModeOff(),
+      ),
+    );
+    maintenanceStatus = await api.query.maintenance.maintenanceStatus();
+    expect(maintenanceStatus.isMaintenance.toString()).toEqual("false");
```

Comment:
perhaps you can move this to the beforeEach?

Reply:
There are other tests in this file. I can create describe fro these three tests or create a new file, I don't know which variant is the best